### PR TITLE
Fix `react/no-unescaped-entities` error

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Fixes `react/no-unescaped-entities` linter error

### CODE REVIEW

File: frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx [hunk 1]
```diff
@@ -268,7 +268,7 @@ export default class SettingsSlackForm extends Component {
           </div>
           <div className="py2">
             {jt`Once you're there, give it a name and click ${(
-              <strong>"Add bot integration"</strong>
+              <strong>&quot;Add bot integration&quot;</strong>
```
Comment (nemanjaglumac) [inline frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx:271]: Will this string be translated?

Comment (daltojohnso) [reply frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx:271]: Looks like this is a missed translation

Comment (daltojohnso) [reply frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx:271]: Wrap it in `t`

**Review by daltojohnso — APPROVED** [review]
Just wrap that one string in `ttag` and it should be good to go.

**Review by daltojohnso — APPROVED** [review]

**Pull request merged**